metrics/core/src/main/java/org/hyperledger/besu/metrics/prometheus/MetricsConfiguration.java

```diff
@@ -121,6 +121,12 @@ public String getPrometheusJob() {
     return prometheusJob;
   }
 
+  /** use getHostsAllowlist instead */
```

Comment:
Make it a non-javadoc comment, otherwise there is a chance that some check will fail in the build.
i.e. `// use getHostsAllowList instead`